config/packages/package-operator/components/remote-phase/manifest.yaml.tpl

```diff
@@ -864,14 +864,14 @@ test:
           name: test
           namespace: test-ns
       config:
-        affinity:
+        subcomponentAffinity:
           nodeAffinity:
             requiredDuringSchedulingIgnoredDuringExecution:
               nodeSelectorTerms:
               - matchExpressions:
                 - key: node-role.kubernetes.io/infra
                   operator: Exists
-        tolerations:
+        subcomponentTolerations:
         - effect: NoSchedule
           key: node-role.kubernetes.io/infra
-    name: affinity-tolerations-resources
+    name: component-affinity-tolerations-resources
```

Comment:
I don't thinkg that the keys should be changed inside the subcomponent package itself.